lib/ansible/modules/storage/netapp/na_ontap_volume.py

```diff
@@ -684,6 +697,8 @@ def create_volume_options(self):
             options['export-policy'] = self.parameters['policy']
         if self.parameters.get('junction_path'):
             options['junction-path'] = self.parameters['junction_path']
+        if self.parameters.get('comment'):
+            options['comment'] = self.parameters['comment']
```

Comment:
this field shows as volume-comment in 9.6 ZAPI documentation.   Is this version dependent?

Reply:
Sorry, I missed that ZAPI discrepancy between the create, modify and get calls. Now fixed.